core/domain/activity_domain.py

```diff
@@ -28,17 +28,30 @@ class ActivityReference(object):
     """
```

Comment:
Have a look at the [style guide](https://google.github.io/styleguide/pyguide.html?showone=Comments#Comments). I think you need to document the attributes, too?